pages/connections/farmlands.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <main>
+    <div
+      class="bg-bottom bg-cover bg-[#174B33] bg-blend-overlay bg-[url('~/assets/images/countryside.jpg')]"
+    >
+      <div class="desktop-gutters md:px-40 py-16 px-4 space-y-6">
+        <header
+          class="space-y-6"
+        >
+          <span
+            class="text-content-on-color text-base leading-6 uppercase font-semibold tracking-wider block"
```

Comment:
@AndyClifford let me know when you ask about the fonts, I'm having a similar issue 👍

Reply:
Junjie said he will make some changes to the partner page design where possible to use our existing fonts - will follow up with him again

Reply:
As discussed we will follow up with using our existing typography classes to maintain consistency